app/views/pages/ineligible.scala.html

```diff
@@ -112,10 +87,10 @@ <h1 class="form-title heading-xlarge" id="pageHeading">@messages("pages.serviceC
     }
 
     @form(action = Call("GET", "https://online.hmrc.gov.uk/registration/newbusiness/business-allowed")) {
-        <div class="form-group">
-            <button type="submit" class="btn button" role="button" id="continue">
-            @messages("pages.serviceCriteriaFailure.button.linkToLegacyApp")
-            </button>
-        </div>
+        <!--<div class="form-group">-->
```

Comment:
Why is all this being commented out?

Reply:
Button is no longer needed on those pages, removing it now. thanks